src/pages/index.tsx

```diff
@@ -87,9 +87,11 @@ export default function Index(): JSX.Element {
           />
         </section>
 
-        <Link to="/learn" className="btn-primary">
-          Get Started
-        </Link>
+        <div className="download-lts-container">
```

Comment:
As another idea/option to center this `Link`, the content for `<div className="home-page">` can be centered and this `div` could be not necessary.

Reply:
That would not be very atomic component like approach. The classes also need to be overhauled and renamed but that's something we can handle in a separate PR. Right now, my priority is to fix the a11y and consistency. 

Later on a button component would suffice here that can have all the attribs as props to handle situations like these I suppose.